website/views/options/social/hall.jade

```diff
@@ -6,6 +6,11 @@ script(type='text/ng-template', id='partials/options.social.hall.html')
     li(ng-class="{ active: $state.includes('options.social.hall.patrons') }")
       a(ui-sref='options.social.hall.patrons')
         =env.t('hallPatrons')
+  ul.options-blurbmenu
+      span(data-ng-show="$state.includes('options.social.hall.heroes')")
```

Comment:
Are these span necessary? Also, why are you using the `data-...` syntax?

This should work:

```jade
ul.options-blurbmenu
  li(ng-show="$state.includes('options.social.hall.heroes')")!=env.t('blurbHallHeroes')
  li(ng-show="$state.includes('options.social.hall.patrons')")=env.t('blurbHallPatrons')
```